tests/data/placebo/test_elb_attribute_filter/elasticloadbalancing.DescribeLoadBalancerAttributes_1.json

```
@@ -0,0 +1,34 @@
+{
+    "status_code": 200, 
+    "data": {
+        "ResponseMetadata": {
+            "RetryAttempts": 0, 
```

ResponseMetadata on these can be an empty dict

```suggestion
        "ResponseMetadata": {}
```